docs/reference.md

```diff
@@ -182,6 +183,34 @@ const asMaximum = asOptional(asNumber)
 asMaximum(null) // returns undefined
 ```
 
+## asValue
```

Comment:
Mention "enum" somewhere in this section for the noobs.

Reply:
Ok, I re-wrote it a bit.